tests/vmi_lifecycle_test.go

```diff
@@ -573,6 +573,42 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 
 				tests.WaitForSuccessfulVMIStart(newVMI)
 			})
+
+			It("[test_id:5275]VMIs with Bridge Networking shouldn't fail after the kubelet restarts", func() {
+				bridgeVMI := vmi
+				// Remove the masquerade interface to use the default bridge one
+				bridgeVMI.Spec.Domain.Devices.Interfaces = nil
+				bridgeVMI.Spec.Networks = nil
+				v1.SetDefaults_NetworkInterface(bridgeVMI)
+				Expect(bridgeVMI.Spec.Domain.Devices.Interfaces).NotTo(BeEmpty())
+
+				By("starting a VMI with bridged network on a node")
+				bridgeVMI, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(bridgeVMI)
+				Expect(err).To(BeNil(), "Should submit VMI successfully")
+
+				// Start a VirtualMachineInstance with bridged networking
+				nodeName := tests.WaitForSuccessfulVMIStart(bridgeVMI)
+
+				tests.VerifyDummyNicForBridgeNetwork(bridgeVMI)
+
+				By("restarting kubelet")
+				pod := renderPkillAllPod("kubelet")
+				pod.Spec.NodeName = nodeName
+				_, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(pod)
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() error {
+					By("starting another VMI on the same node, to verify kubelet is running again")
+					newVMI := newCirrosVMI()
+					newVMI.Spec.NodeSelector = map[string]string{"kubernetes.io/hostname": nodeName}
+					newVMI, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(newVMI)
```

Comment:
I believe putting just the `create` in the `Eventually` should be enough.

Reply:
Changed it.

@AlonaKaplan what do you think is causing the jobs to fail?  Could it be that the kubelet restart is causing something in the network to collapse?
```
 16:07:32:       Unexpected Warning event received: testvmi-tbqht,bf4097c3-9294-4e98-92d1-91934078cba5: failed to configure vmi network: Critical network error: device or resource busy
16:07:32:       Expected
16:07:32:           <string>: Warning
16:07:32:       not to equal
16:07:32:           <string>: Warning 
```

Reply:
Is this warning you post here still relevant?

Reply:
No.  This error was because I was changing something with the interface while it was up